### PR TITLE
add unasync to project dependencies

### PR DESCRIPTION
**Pull request by miguelgrinberg**
Added unasync and isort to pyproject.toml. This was missed in the unasync PR.

### CODE REVIEW

File: libs/elasticsearch/pyproject.toml [hunk 1]
```diff
@@ -58,6 +58,11 @@ optional = true
 [tool.poetry.group.test_integration.dependencies]
 
 
+
+[tool.poetry.group.codegen.dependencies]
+unasync = "^0.6.0"
+isort = "^5.13.2"
```
Comment (pquentin) [inline libs/elasticsearch/pyproject.toml:64]: I did not notice that the last time, but should we remove isort, as ruff includes it already?

**Review by pquentin — APPROVED** [review]
Thanks! LGTM. The isort change can be a follow-up too.

File: libs/elasticsearch/scripts/run_unasync.py [hunk 2]
```diff
@@ -111,7 +111,7 @@ def main(check=False):
     for dir in source_dirs:
         output_dir = f"{dir[0]}_sync_check/" if check else dir[1]
         subprocess.check_call(["ruff", "format", "--target-version=py38", output_dir])
-        subprocess.check_call(["isort", output_dir])
+        subprocess.check_call(["ruff", "check", "--fix", "--select", "I", output_dir])
```
Comment (pquentin) [inline libs/elasticsearch/scripts/run_unasync.py:114]: We already have `--select I` in pyproject.toml.

File: libs/elasticsearch/pyproject.toml (hunk 1 above)
Comment (miguelgrinberg) [reply libs/elasticsearch/pyproject.toml]: I'm just repeating what's in the Makefile:

https://github.com/langchain-ai/langchain-elastic/blob/686575ca13bd9b0936d38a3efed4803b47711f6e/libs/elasticsearch/Makefile#L30-L38

File: libs/elasticsearch/scripts/run_unasync.py (hunk 2 above)
Comment (miguelgrinberg) [reply libs/elasticsearch/scripts/run_unasync.py:114]: I'm just repeating what's in the Makefile:

https://github.com/langchain-ai/langchain-elastic/blob/686575ca13bd9b0936d38a3efed4803b47711f6e/libs/elasticsearch/Makefile#L30-L38

Comment (pquentin) [reply libs/elasticsearch/scripts/run_unasync.py:114]: I suspect it's not needed! But not familiar with ruff to tell.

**Review by pquentin — COMMENTED** [review]
Did the poetry.lock update break the integration tests?

**Comment by miguelgrinberg** [comment]
Yes, indirectly it did, but for a good reason. The tests were designed to work with the <=8.15 Python client, which used knn for hybrid search. We updated the 8.16 client to use a retriever query instead, so now these tests need updating.

**Pull request merged**